pallets/crowdloans/src/lib.rs

```diff
@@ -1368,5 +1497,21 @@ pub mod pallet {
             ));
             Ok(())
         }
+
+        // Returns bool value for all contributions
+        fn has_childstorage(vault: &mut Vault<T>) -> bool {
+            let mut count = 0;
+            for storage_kind in [
+                ChildStorageKind::Contributed,
+                ChildStorageKind::Flying,
+                ChildStorageKind::Pending,
+            ] {
+                count = count
+                    + Self::contribution_iterator(vault.trie_index, storage_kind)
+                        .fold(0u128, |sum, (_account, (amount, _ref_code))| sum + amount);
```

Comment:
`.count` api is better since we dont care about the sum but just the number of contributions